### PR TITLE
docs: adding PR guidelines for contributors

### PR DESCRIPTION
**Pull request by carlsverre**
I recently made my first contribution and through the process learned what was required in each PR.  This PR hopes to help future contributors in following the same guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,3 +25,49 @@
...
+* test: Adding missing tests or correcting existing tests
+* build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+* ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+* chore: Other changes that don't modify src or test files
+* revert: Reverts a previous commit
+
+### Changelog
+
+Add a brief description of your pull request to the changelog located here:
+[src/packages/excalidraw/CHANGELOG.md](src/packages/excalidraw/CHANGELOG.md)
```
Comment (lipis) [inline CONTRIBUTING.md:51]:
```suggestion
[`src/packages/excalidraw/CHANGELOG.md`](src/packages/excalidraw/CHANGELOG.md)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,3 +25,50 @@
 1. Tap on `Fork Sandbox`
 1. Write your code
 1. Commit and PR automatically
+
+## Pull Request Guidelines
```
Comment (dwelle) [inline CONTRIBUTING.md:29]:
```suggestion
## Pull Request Guidelines

Don't worry if you get any of the below wrong, or don't know how to do them. We'll gladly help out.
```

**Comment by lipis** [comment]
We don't have many rules but while we are on it, shall we add the `"proseWrap": "never"` and have the files consistent and better diffs in the future?

`npm run fix` will be needed after that.

/cc @dwelle

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,3 +25,52 @@
...
+
+Don't worry if you get any of the below wrong, or if you don't know how. We'll gladly help out.
+
+### Title
+
+Make sure the title starts with a semantic prefix:
+
+- feat: A new feature
```
Comment (lipis) [inline CONTRIBUTING.md:37]:
```suggestion
- **feat**: A new feature
```

**Comment by dwelle** [comment]
> shall we add the `"proseWrap": "never"` and have the files consistent and better diffs in the future?

:+1:

**Pull request merged**